website/src/pages/docs/prefetching.mdx

```diff
@@ -44,7 +44,7 @@ function App() {
 }
 
 const OtherComponent = loadable(() =>
```

Comment:
You can just remove this, it is not part of this example 😅

Reply:
Hah good call, I've removed the `OtherComponent` from the example. I now see the changes are in the `Infos.preload()`, makes them stand out more.